docs-chef-io/content/inspec/shell.md

```diff
@@ -230,3 +230,43 @@ $ inspec shell --format json -c 'describe file("/Users/test") do it { should exi
   }
 }
 ```
+
+## Running Chef InSpec Shell with inputs 
+
+Input option in shell subcommand would allow to more consistently and easily test and work with controls inside shell.
```

Comment:
This is a bit rough. Perhaps:

The input options for the shell command allow you to provide values to profiles that are parameterized.  The allows you to work more consistently with these profiles when switching between `shell` and `exec` when using profiles that have inputs. For more details on inputs, see [the inputs reference](https://docs.chef.io/inspec/inputs/).